addons/website_sale_mondialrelay/static/src/js/checkout.js

```diff
@@ -53,14 +55,26 @@ websiteSaleCheckoutWidget.include({
      */
     _loadMondialRelayModal: function (result) {
         // add modal to body and bind 'save' button
-        $(renderToElement('website_sale_mondialrelay', {})).appendTo('body');
-        this.$modal_mondialrelay = $('#modal_mondialrelay');
-        this.$modal_mondialrelay.find('#btn_confirm_relay').on('click', this._onClickBtnConfirmRelay.bind(this));
+        document.body.append(renderToElement("website_sale_mondialrelay", {}));
+        this.modal_mondialrelay = document.querySelector("#modal_mondialrelay");
+        this.modal_mondialrelay
+            .querySelector("#btn_confirm_relay")
+            ?.addEventListener("click", this._onClickBtnConfirmRelay.bind(this));
 
         // load mondial relay script
-        const script = document.createElement('script');
-        script.src = "https://widget.mondialrelay.com/parcelshop-picker/jquery.plugin.mondialrelay.parcelshoppicker.min.js";
-        script.onload = () => {
+        const iframeEL = document.createElement("iframe");
+        iframeEL.src = "/website_sale_mondialrelay/get_mondialrelay";
+        iframeEL.style.width = "100%";
+        iframeEL.style.height = "950px";
```

Comment:
There are CSS issues with the modal:
- A vertical scrollbar appears for no reason
![image](https://github.com/user-attachments/assets/8b79b7a3-6532-4bde-bd98-441d016f835d)
- On mobile view the bottom buttons are hidden
![image](https://github.com/user-attachments/assets/2d6f2e91-1e6d-4a53-b355-4cc864a95e32)